semcore/wizard/src/Wizard.jsx

```diff
@@ -142,20 +190,70 @@ function Stepper(props) {
 Stepper.enhance = [keyboardFocusEnhance()];
 
 function Content(props) {
-  const { Children, styles } = props;
+  const { Children, children: hasChildren, styles, uid, step } = props;
   const SContent = Root;
   return sstyled(styles)(
-    <SContent render={Box}>
-      <Children />
+    <SContent
+      render={Box}
+      role='tabpanel'
+      aria-labelledby={`${uid}-step-${step}`}
```

Comment:
it refers to the wrong element (the tabpanel)
it should refer to the current Stepper (tab)